test/steps/global.js

```diff
@@ -1,34 +1,30 @@
 import simulateEvent from "simulate-event";
 
-/*global sinon,expect,StepTest,Traitify*/
-
-function Init(){
-  /*eslint-disable prefer-arrow-callback*/
```

Comment:
Are we not using the linter anymore?

Reply:
Oh yeah another thing I meant to fix was the linter. Right now there were so many errors from it that I had to run the tests directly. I'll add these back in when I get to that